Themis/Views/Assessment/CorrectionSidebar/CorrectionSidebarView.swift

```diff
@@ -12,8 +12,8 @@ enum CorrectionSidebarElements {
 }
 
 struct CorrectionSidebarView: View {
-    @StateObject var feedbackModel = FeedbackViewModel.mock
-    @StateObject var correctionGuidelines = CorrectionGuidelines.mock
+    @StateObject var feedBackViewModel = FeedbackViewModel()
```

Comment:
I think the spelling of `feedback` / `feedBack` should be consistent => `feedbackViewModel`?

Reply:
This is something I didn't do yet. Will completely rework this in the future. So not relevant yet.